Doc/library/dbm.rst

```diff
@@ -264,12 +264,12 @@ to locate the appropriate header file to simplify building this module.
    Name of the ``ndbm`` implementation library used.
 
 
-.. function:: open(filename[, flag[, mode]])
+.. function:: open(filename[, flags[, mode]])
```

Comment:
Double-check me, but https://github.com/python/cpython/blob/master/Modules/_dbmmodule.c#L427 makes this even more accurate:
```suggestion
.. function:: open(filename[, flags="r"[, mode]])
```